packages/alchemy/src/react/components/auth/card/index.tsx

```diff
@@ -63,22 +64,23 @@ export const AuthCard = (
         createPasskeyAfter: urlParams.get(IS_SIGNUP_QP) === "true",
       });
     }
-  }, [authStep, status, props, isAuthenticating, setAuthStep]);
+  }, [authStep, status, isAuthenticating, setAuthStep, onAuthSuccess]);
 
   return (
     <div className="relative">
       <div
         id="akui-default-error-container"
         className="absolute bottom-[calc(100%+8px)] w-full"
       >
-        {!props.hideError && error && error.message && (
+        {!hideError && error && error.message && (
           <Notification message={error.message} type="error" />
         )}
       </div>
       <div className="modal-box relative flex flex-col items-center gap-5 text-fg-primary">
-        {props.showNavigation && (
+        {(canGoBack || showClose) && (
           <Navigation
-            showingBack={canGoBack}
+            showClose={showClose}
+            showBack={canGoBack}
```

Comment:
same comment or question? Is it possible to showClose at the same time as back? 

Same question as `packages/alchemy/src/react/components/navigation.tsx`